deployment/copilot/kafka-broker/manifest.yml

```diff
@@ -0,0 +1,51 @@
+# The manifest for the "kafka-broker" service.
+# Read the full specification for the "Load Balanced Web Service" type at:
+#  https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/
+
+# Your service name will be used in naming your resources like log groups, ECS services, etc.
+name: kafka-broker
+#type: Backend Service
+type: Load Balanced Web Service
+
+## Distribute traffic to your service.
+http: false
+nlb:
+  port: 9092
+  
+# Allow service-to-service communication with ECS Connect
+network:
+  connect: true
+
+# Configuration for your containers and service.
+image:
+  location: docker.io/bitnami/kafka:3.4
+  # Port exposed through your container to route traffic to it.
+  port: 9092
+
+cpu: 256       # Number of CPU units for the task.
+memory: 2048    # Amount of memory in MiB used by the task.
+count: 1       # Number of tasks that should be running in your service.
+exec: true     # Enable running commands in your container.
+
+# storage:
+  # readonly_fs: true       # Limit to read-only access to mounted root filesystems.
+ 
+# Optional fields for more advanced use-cases.
+#
+variables:                    # Pass environment variables as key value pairs.
+  ALLOW_PLAINTEXT_LISTENER: yes
+  KAFKA_ENABLE_KRAFT: no
+  KAFKA_ZOOKEEPER_CONNECT: kafka-zookeeper:2181
+  KAFKA_LISTENERS: PLAINTEXT://:9092
+  KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT
+  KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka-broker:9092
+
+#secrets:                      # Pass secrets from AWS Systems Manager (SSM) Parameter Store.
+#  GITHUB_TOKEN: GITHUB_TOKEN  # The key is the name of the environment variable, the value is the name of the SSM parameter.
+
+# You can override any of the values defined above by environment.
+environments:
+  test:
+    count: 1               # Number of tasks to run for the "test" environment.
+    deployment:            # The deployment strategy for the "test" environment.
+      rolling: 'recreate' # Stops existing tasks before new ones are started for faster deployments.
```

Comment:
I would observe rolling (overlapped) behavior if I manually recycled the container w/ copilot, right?  There's no other auto-bounce present, right?
It's worth pointing out somewhere that w/ only one kafka instance and no EFS, this configuration isn't meant to be durable at all!